provider/datagen/src/transform/cldr/source.rs

```diff
@@ -87,6 +88,14 @@ impl CldrCache {
             },
         )
     }
+
+    #[cfg(feature = "experimental")]
+    pub fn date_fields(&self) -> CldrDirLang<'_> {
```

Comment:
issue: this should just be `.dates("gregorian")`, no?

Reply:
Good catch!
Removing "date_fields".